packages/lib/server/service/replexica.ts

```diff
@@ -0,0 +1,33 @@
+import { ReplexicaEngine } from "@replexica/sdk";
+
+import { REPLEXICA_API_KEY } from "@calcom/lib/constants";
+
+export class ReplexicaService {
+  private static engine = new ReplexicaEngine({
+    apiKey: REPLEXICA_API_KEY,
+  });
+
+  /**
+   * Localizes text from one language to another
+   * @param text The text to localize
+   * @param sourceLocale The source language locale
+   * @param targetLocale The target language locale
+   * @returns The localized text
+   */
+  static async localizeText(text: string, sourceLocale: string, targetLocale: string): Promise<string> {
```

Comment:
Can this take multiple locales so we only call the SDK once?

Reply:
According to the API docs, nope

ref: https://docs.replexica.com/setup/sdk